tests/backends/base/test_creation.py

```diff
@@ -40,3 +41,29 @@ def test_custom_test_name_with_test_prefix(self):
         test_connection.settings_dict['TEST'] = {'NAME': test_name}
         signature = BaseDatabaseCreation(test_connection).test_db_signature()
         self.assertEqual(signature[3], test_name)
+
+
+@mock.patch.object(connection, 'ensure_connection', return_value=None)
```

Comment:
The return value is never checked in any code, so can drop:

```suggestion
@mock.patch.object(connection, 'ensure_connection')
```